frontend/src/components/GoalSettingModal.tsx

```
@@ -0,0 +1,240 @@
+            return [1, 1];
+        }
+    } else if (timePeriod === TimePeriodEnum.Monthly) {
+        if (measure === MeasureEnum.ModulesCompleted) {
+            return [20, 50];
+        } else if (measure === MeasureEnum.GemsEarned) {
+            return [30, 100];
+        } else if (measure === MeasureEnum.ExtendStreak) {
+            return [2, 30];
+        }
+    }
+    return [0, Infinity];
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)